docs/what-is-feldera.md

```diff
@@ -2,7 +2,7 @@
 slug: /
 ---
 
-# What is Feldera?
+# Introduction
```

Comment:
I still think this isn't a very helpful title. I prefer "What is Feldera?"

Reply:
All right, we'll channel our inner Mihai's for this one :)